From 2760768034265780b0049c4ca582348332a51c2b Mon Sep 17 00:00:00 2001
From: Michael Marineau <michael.marineau@coreos.com>
Date: Thu, 27 Nov 2014 14:54:27 -0800
Subject: [PATCH 11/16] gpt: switch partition names to a 16 bit type

In UEFI/GPT strings are UTF-16 so use a uint16 to make dealing with the
string practical.
---
 include/grub/gpt_partition.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/grub/gpt_partition.h b/include/grub/gpt_partition.h
index 166fd4b..1142317 100644
--- a/include/grub/gpt_partition.h
+++ b/include/grub/gpt_partition.h
@@ -92,7 +92,7 @@ struct grub_gpt_partentry
   grub_uint64_t start;
   grub_uint64_t end;
   grub_uint64_t attrib;
-  char name[72];
+  grub_uint16_t name[36];
 } GRUB_PACKED;
 
 enum grub_gpt_part_attr_offset
-- 
2.34.1.windows.1

